src/components/Bounty/BountyContextMenu.tsx

```diff
@@ -57,6 +58,13 @@ export function BountyContextMenu({
         Delete
       </Menu.Item>
     ) : null,
+    isModerator ? (
```

Comment:
Since you have the isModerator in the component itself, you can probably remove this here.